docs/development.md

```diff
@@ -57,6 +57,24 @@ cd foreman
 bundle exec foreman start
 ```
 
+### Stable Katello Development Box
+
+There is a stable box image of a full Katello development install published to Vagrant cloud nightly. In the `vagrant/boxes.d/99-local.example` file, a box using this image is available under the name `centos7-katello-devel-stable`
+
+```
+vagrant up centos7-katello-devel-stable
+```
+
+Once you have spun up the box, to create a new `centos-katello-devel-stable` box using the latest image from vagrant cloud you will need to update the underlying box image. For example:
```

Comment:
This sentence is a bit confusing to me. You are trying to explain how to get a newer stable vagrant box? I would include a note that it will remove the existing environment as a warning rather than "updating" the existing which I think users might perceive.

Reply:
updated, let me know if its not clear now!